.vib/grafana-tempo-vulture/goss/vars.yaml

```diff
@@ -0,0 +1,6 @@
+binaries:
+  - tempo-vulture
+root_dir: /opt/bitnami
+directories:
+  - paths:
+    - /opt/bitnami/grafana-tempo-vulture/bin
```

Comment:
Given that we aren't checking the `tempo-vulture` bin itself and instead doing this dir check, I'll substitute this check with a `/opt/bitnami/grafana-tempo-vulture/bin/tempo-vulture --help`. That way you would both check the dir and the binary.